backend/src/api/masters/masters.api.ts

```diff
@@ -42,6 +45,27 @@ const initMastersApi: FastifyPluginAsync<Options> = async (fastify, opts) => {
       return rep.send(user).status(HttpCode.CREATED);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.POST,
+    url: MastersApiPath.SIGN_IN,
+    schema: {
+      body: masterSignInValidationSchema,
+    },
+    validatorCompiler({
+      schema,
+    }: FastifyRouteSchemaDef<typeof masterSignInValidationSchema>) {
+      return (
+        data: MasterSignInDto,
+      ): ReturnType<typeof masterSignInValidationSchema['validate']> => {
+        return schema.validate(data);
+      };
+    },
+    async handler(req: FastifyRequest<{ Body: MasterSignInDto }>, rep) {
+      const user = await masterService.verifyLoginCredentials(req.body);
+      return rep.send(user).status(HttpCode.OK);
```

Comment:
```suggestion
      const signInUserPayload = await masterService.verifyLoginCredentials(req.body);
      return rep.send(signInUserPayload).status(HttpCode.OK);
```
do save for the sign-up request